keras_nlp/models/bart/bart_tokenizer.py

```diff
@@ -117,3 +120,11 @@ def __init__(
     @classproperty
     def presets(cls):
         return copy.deepcopy(backbone_presets)
+
+    def get_config(self):
+        config = super().get_config()
+        # In the constructor, we pass the list of special tokens to the
+        # `unsplittable_tokens` arg of the superclass' constructor. Hence, we
+        # delete it from the config here.
+        del config["unsplittable_tokens"]
```

Comment:
If we don't remove it from the config, will that just get written again in the `from_config` time?

Reply:
If we don't remove if from the config, if will get passed to the `BartTokenizer` constructor. I was advising the `del` as we don't actually want to support the user providing their own unsplittables at the Bart modeling level. `BartTokenizer` takes over `unsplittable_tokens` as an internal option.